### PR TITLE
Implemented optional XML escaping

### PR DESCRIPTION
**Pull request by doihaveto**
Fixes issue #93

### CODE REVIEW

File: ebaysdk/utils.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 '''
 import sys
 from lxml import etree as ET
+from xml.sax.saxutils import escape
```
Comment (timotheus) [inline ebaysdk/utils.py]: Does xml.sax ship with standard python distributions?

Comment (doihaveto) [inline ebaysdk/utils.py]: It is a standard library as far as I'm aware, yes. https://docs.python.org/2/library/xml.sax.utils.html

**Comment by jmsfwk** [comment]
`StandardError` doesn't exist in Python 3

File: ebaysdk/utils.py [hunk 2]
```diff
@@ -256,6 +257,8 @@ def dict2xml(root):
 
             else:
                 value = root[key]
+                if escape_xml and (isinstance(value, str) or isinstance(value, unicode)) and not value.startswith('<![CDATA['):
```
Comment (jmsfwk) [inline ebaysdk/utils.py]: `unicode` is not defined in Python 3.

**Comment by doihaveto** [comment]
Made the necessary changes for python 3 support

**Pull request merged**